lib/job-simulation.rb

```diff
@@ -1,31 +1,58 @@
-require './Stack.rb'
-require './Queue.rb'
+require_relative 'Stack'
+require_relative 'Queue'
 
 class JobSimulation
-  attr_reader :workers, :waiting, :roll
+  attr_reader :employed, :waiting, :waitlist
 
   def initialize (jobs_available, job_seekers)
+    #job_seekers must be larger than jobs_available
+    @waiting = job_seekers - jobs_available
+    give_jobs(jobs_available)
+    create_wait_list
+  end
 
+  def give_jobs(jobs_available)
+    @number = 1
+    @employed = Stack.new
+    jobs_available.times do
+      @employed.push("Worker \##{@number}")
+      @number += 1
+    end
+    @employed
+  end
+
+  def create_wait_list
+    @waitlist = Queue.new
+    @waiting.times do
+      @waitlist.enqueue("Worker \##{@number}")
+      @number += 1
+    end
+    @waitlist
```

Comment:
same note about style of loop and using `|i|` here. Though this one is a little trickier to make sure that the numbering would start at 7, in this case, instead of 1. But it can be done with a little math.